Spanner/src/Transaction.php

```diff
@@ -739,6 +740,6 @@ private function buildUpdateOptions(array $options): array
         $selector = $this->transactionSelector($options);
         $options['transaction'] = $selector[0];
 
-        return $options;
+        return $this->addLarHeader($options);
```

Comment:
Good opportunity to add snippets in some of these methods and then add snippet tests to those as well.

Definitely in the ones that need this header unconditionally.

Reply:
Default is to add this header to all the supported operations internally. The only place the user can override is via the client. I can add a snippet there, but do you think it is required as it is already mentioned in the docblock of the constructor and its clear to the user?

Also, Spanner team don't want it to be super highlighted as well. Its kind of a hidden feature on the client library and customers may use only if some regression happens.

Reply:
Okay, will add it in the `SpannerClient` then, thats the only place we can opt-out on this.

Reply:
Added